pytorch_lightning/trainer/trainer_io.py

```diff
@@ -93,6 +96,18 @@ def restore_state_if_checkpoint_exists(self, model):
 
         return did_restore
 
+    def restore_state_from_checkpoint(self, checkpoint_path):
+        did_restore = False
```

Comment:
add doc what data type the `checkpoint_path` is

Reply:
I removed this function. Please review the updated code :)